index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <h1 id="logo">Aziz Mejri</h1>
+    <nav id="desktop-nav">
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <ul>
+        <li><a class="nav-link" href="#projects">Portoflio</a></li>
+        <li><a class="nav-link" href="#about-me">About</a></li>
+        <li>
+          <a class="nav-link" href="#contact">Contact <i class="far fa-envelope"></i></a>
```

Comment:
- [ ] [REQUIRED] Please, Implement the changes the previous CR requested.  You can check the requirement list [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md). 
- The mobile menu option should disappear once it's clicked.
- menu links should navigate to the correct path on the page